### PR TITLE
Added support for liquidity RPC calls

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Removed the old way to get liquidityPrice via derived query and update the existing test case to use rpc call.

### CODE REVIEW

File: packages/types/src/extrinsic/types.ts [hunk 1]
```diff
@@ -105,5 +105,7 @@ declare module '@polkadot/types/interfaceRegistry' {
     FeeExchange: FeeExchange;
     'Option<FeeExchange>': Option<FeeExchange>;
     'Vec<(AssetId, AssetInfo)>': Vec<ITuple<[AssetId, AssetInfo]>>;
+    '(Balance, Balance)': '(Balance, Balance)';
```
Comment (holygits) [inline packages/types/src/extrinsic/types.ts:108]: This continues to confuse me 🤔

Comment (KarishmaBothara) [reply packages/types/src/extrinsic/types.ts]: Managed to fix this.. with any.. have removed it from types :)

File: packages/api/src/rpc/cennzx.ts [hunk 2]
```diff
@@ -25,9 +25,25 @@ const sellPrice: RpcMethodOpt = {
   type: 'Balance',
 };
 
+const liquidityPrice: RpcMethodOpt = {
+  description: 'Retrieves the price of liquidity for a particular `asset_id`',
```
Comment (holygits) [inline packages/api/src/rpc/cennzx.ts:29]: nit: "Get the price of liquidity for the given asset ID"

File: packages/api/src/rpc/cennzx.ts [hunk 3]
```diff
@@ -25,9 +25,25 @@ const sellPrice: RpcMethodOpt = {
...
+  isOptional: true,
+  params: [createParam('AssetId', 'AssetId'), createParam('liquidityToBuy', 'Balance')],
+  type: '(Balance, Balance)' as any,
+};
+
+const liquidityValue: RpcMethodOpt = {
+  description: 'Retrieves the value of liquidity in the exchange for `asset_id` for `account`',
```
Comment (holygits) [inline packages/api/src/rpc/cennzx.ts:36]: nit: Get the value of an account's liquidity for the given asset

**Pull request merged**